cli/src/athenapdf.js

```diff
@@ -171,6 +176,24 @@ app.on("ready", () => {
     bw.loadURL(uriArg, loadOpts);
 
     ses = bw.webContents.session;
+
+    // When the --request-header is set, add request headers
+    if (athena.requestHeader && athena.requestHeader !== []) {
```

Comment:
Assuming we are using the above changes, then it's just a matter of checking that the dictionary / map is not empty:

```
Object.keys(athena.requestHeader).length
```